### ISSUE TITLE
Cross posting

### ISSUE THREAD

**Issue by jmarkgraf** [mberneaud/PairAssignment3#5]
Hi Malte,

I think I somehow lost track on what you are working at the moment. From the PDF I see that you started to include graphs for the descriptive statistics for the Sparkassen data - actually something that I thought is my responsibility and on what I am working at the moment.
Who of us is then working on the descriptive statistics for the Election data?
Is the Election data ready for the regression analysis?

Here is how I understood it: I do everything related to Sparkassen data, you do everything related to Election data in terms of data preparation. I do descriptive statistics for Sparkassen data, you do descriptive statistics for Election data (including all the related writing). I do the section on the merging part and the regression analysis once everything is ready.
This is a lot to be done before 6pm. So, let's get straight who is doing what!

**Comment by mberneaud** [comment]
The graphs for the graphs I included in the RMd document was the code which you already wrote and which I pasted over into the Rmd. Once again, this is because the requirement of the assignment states that we should have our graphs be dynamically generated within the Rmd, so it makes sense to have them be created right where they are shown. 

For that reason I suggest that we both work on that same document do our commits and whoever does is the last to push a commit will automatically merge the two documents together. If there are no conflicting lines, this should work without issue, if there are, merge conflicts are relatively easy to resolve, as long as they don't occur too often. 

I am d'accord with the division of labour you proposed, I'll continue on the election data now, doing some descriptive statistics and graphs.

**Comment by jmarkgraf** [comment]
Yep, I would just not include all graphs. I'm done with the first draft of the descriptive statistics section on Sparkassen data.
Somehow, the document can't be knitted again, however. I'll have a look after lunch.

**Comment by mberneaud** [comment]
In order for it to work, you need to set the working directory to be the root directory of the Git repo (wherever you have them saved on your hard drive). Otherwise, R won't find the files in the correct place.

**Comment by jmarkgraf** [comment]
I did that, but the code doesn't run for me. I changed your variable names for the Sparkassen dataset back to my names, because otherwise I was forced to change my entire R code. If I run the different code chunks separately it works, but not as a whole. Could you sync and try yourself? Thanks!

**Comment by mberneaud** [comment]
I pulled your changes, merged them in with mine and after deleting my cache and removing the caching option, it works again now. 

Deleting the cache was what did the trick for me. If you pull my most recent commit now, it should work. 

I'll be out for the day now and will get back to this in the late afternoon / evening if possible.

**Comment by jmarkgraf** [comment]
I'll try that. Thanks! Have you also tried to knit a word document? 
And no stress! Since the deadline got postponed, we are not under time pressure. I will write the part about the regressions without saying anything about the regression outputs as we run the regressions later.

**Comment by jmarkgraf** [comment]
I had a quick look at your descriptive statistics: Keep in mind that you need to first subset the data frame to the period of interest (2000-14). A few suggestions for interesting summary statistics: Number of overall elections; number of elections that required run-off election; density plot for municipality size; plotting vote shares of winners; share of female mayors (as done by you). These are the variables that we will use for the regressions, thus, it might make sense to talk a bit about those!

**Comment by mberneaud** [comment]
I just had a look at the document statistics of the Rmd we are writing on and so far, we have >1400 words already. Considering this, I'll include the summary statistics you proposed, in a concise manner, but we'll probably have to shorten the document in the end in order to meet the maximum word count. 

As for the period of interest: Is there a particular reasoning behind the decision to include such a wide time-frame into the analysis, when the data set on banks covers a time period from 2006 to 2015? I am sure you and Guillermo have talked about this and there's a reasoning behind it. If you could give me some pointers, I'd mention the reasons in the Rmd.

**Comment by jmarkgraf** [comment]
I saw that too. I think we can definitely cut on the first part.
The the second point: You are right. We should actually focus on the elections between 2006 and 2014. As we at the moment however only have data for Bavaria our case number is low. Therefore I included elections for the period from before. Since we do not match by mayor-year observation but just by name of the mayor, we anyways assume that a mayor gets a seat either immediately after being elected for the first time or holds this seat until she's unseated. Thus, if a person has a seat between 2006 and 2015 that got first elected in 1998, we would assume that the election of interest would be the second one, i.e. in 2004. Thus, a longer time window gives us more observations and, if at all, the estimated effect should be an underestimate. But there is no good reason to take 2000 as a benchmark date.
